docs/tutorials/local_iteration_series/testing_with_artificial_conditions.md

```diff
@@ -150,7 +150,19 @@ The solutions described below share some common features:
 
 There are some inherent limitations that come with the system-wide level of application of these tools - they are not ideal for local testing of more than two peers because we can't have peers with different network conditions between them (if that is important for the test ofcourse).
 
-### Clumsy on Windows
+:::important
+
+Some consoles offer similar functionality at the native SDK level.  Check their documentation for details.
+
+:::
+
+### Clumsy (Windows)
+
+:::note
+
+dummynet is another option that can be used on Windows, however there are known issues when running it on Windows 10 related to signed driver enforcement setting. As such we recommend Clumsy as a good default option on Windows.
```

Comment:
I see "recommend" in multiple places in this doc. @chrispope @Briancoughlin should we really endorse these?
I'm thinking more "we've used these in our own development".

Reply:
I would suggest we author this in a way that brings awareness of third-party tools, but we shouldn't endorse or recommend any in particular. 

No harm in saying this is what we have used during our development.

I would prefer we come across impartial and users can make their own assessments on the tools which best suit them.

Reply:
agreed, rephrased